src/pages/MainPage/components/ProceedContainer/components/SumaryContainer/components/Shipping/index.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { useSelector } from "react-redux";
+
+import { selectBasket } from "features/basket/basketSlice";
+import * as Styles from "./styles";
+
+const Shipping = () => {
+  const basket = useSelector(selectBasket);
+
+  return (
+    <Styles.Wrapper>
+      <span>Shipping</span>
+      <span>{`$${basket.shipping.toFixed(2)}`}</span>
```

Comment:
zrobiłbym selector selectBasketShippingValue i w nim od razu basket.shipping.toFixed(2) - w ten sposob trzymam wiecej logiki w store. Komponent powinien być jak najbardziej dumb :)

Reply:
Super podejście nigdy bym na to nie wpadł :) 
Done.